frontend/projects/dipa-frontend/src/app/modules/overview/archived-projects/archived-projects.component.ts

```diff
@@ -1,48 +1,57 @@
-import { Component, OnDestroy, OnInit, QueryList, ViewChildren } from '@angular/core';
-
+import { Component, OnDestroy, OnInit, AfterViewInit, QueryList, ViewChildren } from '@angular/core';
 import { IncrementsService, MilestonesService, TasksService, Timeline, TimelinesService } from 'dipa-api-client';
 import { ChartComponent } from '../../gantt/chart/chart.component';
-import { forkJoin, Observable, Subscription } from 'rxjs';
+import { forkJoin, Observable, Subscription, combineLatest } from 'rxjs';
 import { map } from 'rxjs/operators';
+import { ActivatedRoute } from '@angular/router';
 
 @Component({
   selector: 'app-archived-projects',
   templateUrl: './archived-projects.component.html',
   styleUrls: ['./archived-projects.component.scss'],
 })
-export class ArchivedProjectsComponent implements OnInit, OnDestroy {
+export class ArchivedProjectsComponent implements OnInit, OnDestroy, AfterViewInit {
   @ViewChildren('charts') charts: QueryList<ChartComponent>;
 
-  timelinesSubscription: Subscription;
-
-  timelineData: Timeline[];
+  public timelineData: Timeline[];
 
-  vmAll$: Observable<any>;
-  periodStartDateSubscription: Subscription;
+  public vmAll$: Observable<any>;
 
-  observablesList: Array<any> = [];
+  public observablesList: Array<any> = [];
 
-  loading: boolean;
+  public loading: boolean;
 
-  periodStartDate = new Date(2020, 0, 1);
-  periodEndDate = new Date(2020, 11, 31);
+  public periodStartDate = new Date(2020, 0, 1);
+  public periodEndDate = new Date(2020, 11, 31);
+  private timelinesSubscription: Subscription;
+  private periodStartDateSubscription: Subscription;
 
-  constructor(
+  public constructor(
     private timelinesService: TimelinesService,
     private milestonesService: MilestonesService,
     private tasksService: TasksService,
-    private incrementsService: IncrementsService
+    private incrementsService: IncrementsService,
+    private route: ActivatedRoute
   ) {}
 
-  ngOnDestroy(): void {
+  public ngOnDestroy(): void {
     this.timelinesSubscription?.unsubscribe();
   }
 
-  ngOnInit(): void {
+  public ngOnInit(): void {
     this.loadTimelines();
   }
 
-  setData(timelineId: number): Observable<any> {
+  public ngAfterViewInit(): void {
+    const joinStream = combineLatest(this.charts.changes, this.route.fragment);
```

Comment:
in dieser Form ist combineLatest deprecated... besser:
const joinStream = combineLatest([this.charts.changes, this.route.fragment]);